ngraph/core/include/ngraph/function.hpp

```diff
@@ -138,6 +139,24 @@ namespace ngraph
         bool evaluate(const HostTensorVector& output_tensors,
                       const HostTensorVector& input_tensors) const;
 
+        /// Return a list of function's side outputs
+        const SinkVector& get_sinks(void) const { return m_sinks; }
+        /// \brief Add new sink nodes to the list to avoid removing
+        /// \param sinks new sink nodes
+        void add_sinks(const SinkVector& sinks);
+
+        /// \brief Delete sink node from the list of sinks
+        /// \param sink Sink to delete
```

Comment:
Please notice in the comment that we don't delete this node from the graph, only from the list. Is that right?

Reply:
Right, will add to comment